tools/paconn-cli/paconn.pyproj

```diff
@@ -73,7 +73,9 @@
     <Folder Include="paconn\common\" />
   </ItemGroup>
   <ItemGroup>
-    <Content Include="paconn\config\cli.flake8" />
+    <Content Include="paconn\config\cli.flake8">
+      <Publish>True</Publish>
```

Comment:
We don't want to publish flake8 files. Can you please elaborate why you think otherwise?